### PR TITLE
sets up skeleton for pmwww

### PR DESCRIPTION
**Pull request by vivex**
https://github.com/shippablepmtool/pmwww/issues/1

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM shipimg/appbase:latest
+
+ADD . /home/shippable/pmwww
+RUN mkdir -p /home/shippable/pmwww/logs
+
+RUN cd /home/shippable/pmwww && npm install
+RUN cd /home/shippable/pmwww && grunt build
+
+ENTRYPOINT ["/home/shippable/pmwww/boot.sh"]
+EXPOSE 31001
```
Comment (pranaypareek) [inline Dockerfile]: Need a new line at the end of everyfile.

File: src/issue-tracker/issueTrackerCtrl.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+(function () {
+  'use strict';
+  pmWww.config(['$routeProvider', function ($routeProvider) {
+      $routeProvider.when('/', {
+        templateUrl: "src/issue-tracker/issue-tracker.html", //always going to load from compiled templates
```
Comment (pranaypareek) [inline src/issue-tracker/issueTrackerCtrl.js]: We should follow the standards just like our main code base.

This line exceeds 80 chars.

File: Dockerfile (hunk 1 above)
Comment (vivex) [inline Dockerfile]: added.

File: shippable.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  post_ci:
+    - if [ "$IS_PULL_REQUEST" != true ]; then sudo docker push $IMAGE_NAME:$BRANCH.$SHIPPABLE_BUILD_NUMBER; fi
+
+integrations:
+  hub:
+    - integrationName: "shipDH"
+      type: docker
+      branches:
+          only:
```
Comment (pranaypareek) [inline shippable.yml]: 2 space indentations

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```
Comment (pranaypareek) [inline public/index.html]: 2 space indentations for the entire file

Comment (vivex) [inline public/index.html]: added

**Pull request merged**